docs/advanced/settings.md

```diff
@@ -503,6 +503,61 @@ This setting can only be used if the `upgrade-checker-enabled` setting is set to
 https://your.upgrade.checker-url/v99/checkupgrade
 ```
 
+### `upgrade-config`
+
+**Definition**: Upgrade-related configuration.
+
+**Default value**: `{"imagePreloadOption":{"strategy":{"type":"sequential"}}}`
+
+**Example**:
+
+```json
+{
+  "imagePreloadOption": {
+    "strategy": {
+      "type": "parallel",
+      "concurrency": 2
+    }
+  }
+}
+```
+
+#### `imagePreloadOption`
+
+**Definition**: Options for the image-preloading phase.
+
```

Comment:
Add two more lines:

Like the new installation, Harvester tends to load all the pre-packed container images in the Harvester ISO to each node. When workloads are scheduled to any node (except the witness node), the container images are ready to use. 

This option provides more flexibilities for different scenairos.